types/puppeteer-screenshot-tester/index.d.ts

```diff
@@ -0,0 +1,314 @@
+import { Page, ScreenshotOptions as PuppeteerScreenshotOptions } from "puppeteer";
+
+interface OutputSettingsPng {
+    forceExt?: null | ".png";
+    compressionLevel?: null | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9;
+    overrides?: {};
+}
+interface OutputSettingsOthers {
+    forceExt?: null | ".jpg" | ".jpeg" | "webp";
+    compressionLevel?: null | OneTo100;
+    overrides?: {};
+}
+
+declare namespace ScreenshotTester {
+    type Rectangle = [x: number, y: number, width: number, height: number];
+    interface MatchingBox {
+        ignoreRectangles?: Rectangle[];
+        includeRectangles?: Rectangle[];
+    }
+
+    type OutputSettings = OutputSettingsPng | OutputSettingsOthers;
+
+    interface ErrorSettings {
+        errorColor?: {
+            red?: ZeroTo255;
```

Comment:
I think this is overdoing it, personally. Kind of annoying that something like `red: Math.round(255 * brightnessScaling)` would cause a type error even if you're _really_ sure `brightnessScaling` is between 0 and 1.

Reply:
Thanks for your feedback!

I get the issue. Something in my mind was telling me that I was overdoing it and I didn't thought that any calculation will result in "number" type.

I will switch ZeroTo255 to number, and leave that information as a comment.

As an extension to this, should I do the same with the compression level?. Is there a use case where someone needs to calculate the compression level?

thanks again